app/assets/stylesheets/application.scss

```diff
@@ -71,7 +87,9 @@ body h2 {
     height: 550px;
     background-image: url(mainback2.jpg);
 }
-
+.border:hover {
+    transform: translate3D(0,-1px,0) scale(1.03);
```

Comment:
I like that you are using translate3D where did you get this from?

Reply:
I had used this styling in one of my project actually . Like When he hovers on a card it should look big or something like slight pop-out . I guess i got this when i was web surfing about this feature !